client/src/pages/CalendarPage/CalendarPage.module.css

```diff
@@ -0,0 +1,53 @@
+.bg {
+  background-image: linear-gradient(
+    #032567 30%,
+    rgba(195, 221, 253, 30) 60%,
+    #032a5c
+  );
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.calendarHeader {
+  color: white;
+  font-weight: 600;
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1.875rem;
+  margin-left: 25px;
+  text-decoration: underline;
+}
+
+.filterButton {
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1.125rem;
+  font-weight: 600;
+  background: transparent;
+  border: none;
+  color: #dadada;
+}
+
+.flexContainer {
+  display: flex;
+  justify-content: flex-end;
+  margin: 10px 20px 15px 0px;
+}
+
+ul {
+  margin: 10px 25px;
+  padding: 0px;
+  overflow: hidden;
```

Comment:
I might be wrong but this should be `overflow-x` no? (Since just using `overflow` should be both covering the x and y axis)

Reply:
That was the intention. `overflow-y` is to prevent the vertical scrolling from appearing. `overflow-x` shouldn't be there too since, at least for mobile, I didn't want any sort of scrolling on the x-axis. I will fix any styles if we ever overflow.